src/test/java/tools/jackson/databind/json/JsonMapperBuilderTest.java

```diff
@@ -0,0 +1,37 @@
+package tools.jackson.databind.json;
+
+import org.junit.jupiter.api.Test;
+import tools.jackson.core.StreamReadFeature;
+import tools.jackson.core.json.JsonFactory;
+import tools.jackson.core.json.JsonWriteFeature;
+import tools.jackson.databind.DeserializationFeature;
+import tools.jackson.databind.ObjectMapper;
+import tools.jackson.databind.SerializationFeature;
+import tools.jackson.databind.testutil.DatabindTestUtil;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+// Test(s) to verify behaviors in JsonMapper.Builder
+public class JsonMapperBuilderTest extends DatabindTestUtil
+{
+
+    @Test
+    public void testBuilderWithJackson2Defaults()
+    {
+        ObjectMapper mapper = JsonMapper.builderWithJackson2Defaults().build();
+        JsonFactory jsonFactory = (JsonFactory) mapper.tokenStreamFactory();
+        assertFalse(mapper.isEnabled(StreamReadFeature.USE_FAST_DOUBLE_PARSER));
```

Comment:
Should also not verify streaming-level setting at databind level.

Reply:
Yes, but it must be done at correct level -- streaming at streaming, databind at databind. Not with overlap.

Reply:
JsonMapper.builderWithJackson2Defaults() is a high level method that creates a factory for you.

I have modified the lower level configureForJackson2 methods not to set factory level configs.